jenkinsfile-runner-tests/tests.sh

```diff
@@ -1,7 +1,11 @@
-#!/bin/bash
+#!/usr/local/bin/bash
```

Comment:
Be careful because this won't work in CI.

Reply:
Actually it won't work at all in CI.  :-)